packages/core/src/projections/component-tree-changes-info/collected-component-tree-change-info.interface.ts

```diff
@@ -0,0 +1,8 @@
+import { AuguryEvent } from '../../events';
+import { ComponentTreeNode } from '../../probes/types/component-tree-node';
+
+export interface CollectedComponentTreeChangeInfo {
```

Comment:
Could this just be simply `ComponentTreeChanges`?

Reply:
well, they aren't the changes... it's collecting data for the changes :/ It's a type we are using to just store results in. Collecting data. We already have component tree changes which is actually the result type.

Reply:
Perhaps a quick summary comment on the type would help clarify.